app/views/devise/sessions/new.html.erb

```diff
@@ -2,26 +2,29 @@
 	<h2>Sign In</h2>
 </div>
 <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+
+  <%= render 'shared/flash' %>
```

Comment:
You're using double quotes in other parts of the system, use it here too to keep it consistent.

Reply:
Changed.